### DISCUSSION TITLE
加载TableCls modeltype=yolo或yolox时出现如下问题

### DISCUSSION THREAD

**Discussion by enymfc25173** [RapidAI/TableStructureRec#79] [Q&A]
onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Load model from /Users/xxx/Projects/MLE/Doc-AI/venv/lib/python3.9/site-packages/table_cls/models/yolo_cls.onnx failed:/Users/runner/work/1/s/onnxruntime/core/graph/model.cc:180 onnxruntime::Model::Model(ModelProto &&, const PathString &, const IOnnxRuntimeOpSchemaRegistryList *, const logging::Logger &, const ModelOptions &) Unsupported model IR version: 10, max supported IR version: 9

Error loading: /Applications/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevd_attach_to_process/attach_x86_64.dylib

Process finished with exit code 130 (interrupted by signal 2: SIGINT)

**Comment by enymfc25173** [comment]
python version: 3.9
环境：MacOS

**Reply by Joker1212** [reply]
请问是安装的最新的onnxruntime版本吗？怀疑是版本问题，另外因为默认就是yolo的模型，所以可以不指定试试呢？

**Reply by enymfc25173** [reply]
table-cls包的版本是1.14
如果不指定是相同的报错

**Reply by enymfc25173** [reply]
linux环境结果一样
都是在python的venv中运行的

**Reply by Joker1212** [reply]
环境都是py 3.9吗？可能是环境版本低了，暂时使用qanything那个table-cls呢？输入'q'作为参数就行了

**Reply by Joker1212** [reply]
大多数情况下效果差不多，在那种左右有竖线，上下无竖线的场景，那个模型会误判为有线表格
